docs/glossary/README.md

```diff
@@ -0,0 +1,287 @@
+- [Read this blog article on how to create an intelligent workflow with custom action - explained with an example](https://www.openproject.org/blog/customise-workflows/)
+
+## Custom field
+
+In OpenProject, a custom field is defined as an additional value field which can be added to existing value fields. The different sections that can use custom fields are *work packages*, *spent time*, *projects*, *versions*, *users*, *groups*, *activities (time tracking)*, and *work package priorities*.
+
+** More information on custom fields in OpenProject**
+
+- [Read how to enable custom fields in projects to use them in work packages](https://www.openproject.org/docs/user-guide/projects/project-settings/custom-fields/)
+- 
+
+## Custom query
+
+A custom query in OpenProject consists of saved filters and groupings in the work package list. Custom queries can be either set to public (visible by any user who is allowed to see the project and the work package list) or to private (visible only to the person creating the query).
+
+## Dashboard
+
+A dashboard is defined as an overview page in a software. In OpenProject, you have several options to create dashboards:
+
+First, you have the "***My Page\***" which shows your personal customized widgets on one page, for example a calendar or work package reports.
+
+Then, you can create **private dashboards inside your project** by filtering a work package view and then saving it under a new name. For example, filter all work packages assigned to yourself and save this view as "assigned to me", to quickly navigate to those work packages.
+
+Admins of a project can also create **dashboards that are visible to everyone in the project**.
+
+## Date alerts
+
+Date alerts in OpenProject are an Enterprise add-on and defined as a feature to generate automatic and customized notifications regarding a work package's due date or start date. You can find the date alerts feature in your notification center, symbolized by a little bell on the right upper side of your instance. [Read more about the date alerts feature in our user guide](https://www.openproject.org/docs/user-guide/notifications/notification-settings/#date-alerts-enterprise-add-on) or in [this article on deadline management with OpenProject](https://www.openproject.org/blog/deadline-management-date-alert/)
+
+## Enterprise add-on
+
+In OpenProject, some features are defined as an Enterprise add-on. This means, they are not part of the Community edition and therefore not free of charge. Enterprise add-ons are available as Cloud or on-premises versions. You can chose from the following plans to get access to all Enterprise add-ons: Basic, Professional, Premium and Corporate. [Read more about OpenProject plans and pricing](https://www.openproject.org/pricing/)
+
+## Effort (shirt size)
+
+## Excel synchronisation
+
+https://www.openproject.org/docs/system-admin-guide/integrations/excel-synchronization/
```

Comment:
Please always add a link title to avoid having full URLs printed to HTML.

e.g.
`
[Read more about Excel synchronization with OpenProject](https://www.openproject.org/docs/system-admin-guide/integrations/excel-synchronization/)
`

Reply:
Yes, this URL was only a note to myself to remember the link. Will update this soon!